### PR TITLE
Update MiDaS models

### PR DESCRIPTION
**Pull request by ranftlr**
MiDaS received a major update. This PR updates hub usage instructions accordingly.

### CODE REVIEW

File: intelisl_midas_v2.md [hunk 1]
```diff
@@ -15,25 +15,18 @@ featured_image_2: no-image
...
-multi-objective optimization to ensure high quality on a wide range of inputs.
-
+[MiDaS](https://arxiv.org/abs/1907.01341) computes relative inverse depth from a single image. The repository provides multiple models that cover different use cases ranging from a small, high-speed model to a very large model that provide the highest accuracy. The models have been trained on 10 distinct dataset using 
+multi-objective optimization to ensure high quality on a wide range of inputs. An overview of 
 
 ### Example Usage
 
+Install dependencies
+```python
+!pip install timm
```
Comment (ailzhang) [inline intelisl_midas_v2.md:27]: This won't work. Please move it to https://github.com/pytorch/hub/blob/master/scripts/install.sh :D

**Review by ailzhang — APPROVED** [review]
LGTM! Thanks for the update!

**Comment by ranftlr** [comment]
Fixed. timm is already in the install script. Is timm available in the generated Colab notebook too?

**Comment by ailzhang** [comment]
@ranftlr for colab notebook, would you mind sending a PR to https://github.com/pytorch/pytorch.github.io/blob/master/assets/hub/intelisl_midas_v2.ipynb ? Thanks!

**Comment by ranftlr** [comment]
@ailzhang Thanks for merging. The shell command for installing timm actually works in the colab notebook, no "!" required.

**Pull request merged**